client/components/domains/register-domain-step/index.jsx

```diff
@@ -55,20 +57,25 @@ function processSearchQueue() {
 					continue outerLoop;
 				}
 			}
-			reportSearch( queue[ i ] );
+			reportSearchStats( queue[ i ] );
 		}
 }
 
-function reportSearch( { query, section } ) {
-	analytics.recordEvent( 'searchFormSubmit', query, section );
+function reportSearchStats( { query, section, timestamp } ) {
+	let timeDiffFromLastSearchInSeconds = 0;
+	if ( lastSearchTimestamp ) {
+		timeDiffFromLastSearchInSeconds = Math.floor( ( timestamp.valueOf() - lastSearchTimestamp.valueOf() ) / 1000 );
+	}
+	lastSearchTimestamp = timestamp;
+	analytics.recordEvent( 'searchFormSubmit', query, section, timeDiffFromLastSearchInSeconds, ++searchCount );
```

Comment:
I'm not a fan of this style of incrementing the value of a variable - it's very easy to miss `++searchCount` here. IMHO, readability and maintainability is more important here than trying to keep the code as compact as possible. Especially since I usually "ignore" analytics calls when skimming code ;/